frontend/src/components/organisms/LoginForm.tsx

```diff
@@ -94,89 +93,70 @@ const LoginForm = () => {
   // State variables for the notification popups
   const [notifOpen, setNotifOpen] = useState(false);
 
-  const LoginErrorComponent = (): JSX.Element | null => {
-    setNotifOpen(true);
-    return signInErrors ? (
-      <Snackbar
-        variety="error"
-        open={notifOpen}
-        onClose={() => setNotifOpen(false)}
-      >
-        Error: {handleErrors(signInErrors.code)}
-      </Snackbar>
-    ) : null;
-  };
-
-  const handleGoogleLogin = async () => {
-    setIsLoading(true);
-    try {
-      await signInWithGoogle();
-      if (googleUser) {
-        router.push("/events/view");
-      }
-    } catch (err) {}
-    setIsLoading(false);
-  };
-
   const handleLogin: SubmitHandler<FormValues> = async (data) => {
-    setIsLoading(true);
     const { email, password } = data;
-
     try {
       await signInWithEmailAndPassword(email, password);
       if (signedInUser) {
         router.push("/events/view");
       }
     } catch (err) {}
-    setIsLoading(false);
+    setNotifOpen(true);
   };
   return (
-    <div className="space-y-4 ">
-      <LoginErrorComponent />
-      <img src="/lfbi_logo.png" className="w-24" />
-      <form onSubmit={handleSubmit(handleLogin)} className="space-y-4 ">
-        <div className="font-bold text-3xl"> Log In </div>
-        <div>
+    <>
+      {/* Login error component */}
+      <Snackbar
+        variety="error"
+        open={notifOpen}
+        onClose={() => setNotifOpen(false)}
+      >
+        Error: {handleErrors(signInErrors?.code)}
+      </Snackbar>
+
+      {/* Login form */}
+      <div className="space-y-4">
+        <img src="/lfbi_logo.png" className="w-24" />
+        <form onSubmit={handleSubmit(handleLogin)} className="space-y-4">
+          <div className="font-bold text-3xl"> Log In </div>
```

Comment:
Moved `LoginErrorComponent` down here because having it as its own component was breaking the `Snackbar` close animation. Also changed `Snackbar` to disable clickaway (so it's harder to accidentally dismiss the notification)

Reply:
Also didn't put `Snackbar` inside the div with the form because doing so creates a visual glitch whenever the Snackbar is activated.

Reply:
Also refactored the notifications everywhere else, including `ProfileForm`, `EventForm`, etc.

Reply:
sounds good